counterpartylib/lib/messages/broadcast.py

```diff
@@ -186,16 +190,17 @@ def parse (db, tx, message):
         cursor.close()
         return
 
+    # stop processing if broadcast is invalid for any reason,
```

Comment:
my consensus hashes are fine with only the `util.enabled('broadcast_invalid_check')` check and removing the `util.enabled('max_fee_fraction'))`